docs/framework/migration-guide/migrating-from-the-net-framework-1-1.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Migrating from the .NET Framework 1.1"
+description: Learn about the steps required to run an application compiled using the .NET Framework 1.1 on Windows 7 or later.
```

Comment:
```suggestion
description: Learn about the steps required to run an application compiled using .NET Framework 1.1 on Windows 7 or later.
```

Reply:
I know it uses "the .NET Framework" in the rest of this article, but it's actually correct to omit it.